feconf.py

```diff
@@ -433,8 +433,12 @@ def get_empty_ratings():
 FEEDBACK_THREAD_URL_PREFIX = '/threadhandler'
 FEEDBACK_THREADLIST_URL_PREFIX = '/threadlisthandler'
 FEEDBACK_MESSAGE_EMAIL_HANDLER_URL = (
-    '%s/email/feedbackemailhandler' % TASKQUEUE_URL_PREFIX)
+    '%s/email/batchfeedbackmessageemailhandler' % TASKQUEUE_URL_PREFIX)
 FEEDBACK_THREAD_VIEW_EVENT_URL = '/feedbackhandler/thread_view_event'
+FEEDBACK_THREAD_STATUS_CHANGE_EMAIL = (
```

Comment:
The name of this constant should end with _URL?

FEEDBACK_STATUS_EMAIL_HANDLER_URL perhaps